.vscode/.mypy.ini

```diff
@@ -1,7 +1,7 @@
 # Global options:
 
 [mypy]
-python_version = 3.10
+python_version = 3.11
 warn_unused_configs = True
 disallow_untyped_defs = True
 pretty = True
```

Comment:
could you replace the previous calls to `from typing_extensions import Self` ?

Reply:
Done